lib/pbench/test/unit/server/database/test_datasets_metadata_db.py

```diff
@@ -415,11 +424,9 @@ def test_mutable_server(self, server_config, attach_dataset):
         ],
     )
     def test_mutable_server_bad(self, server_config, attach_dataset, value, message):
-        """
-        Try out some invalid deletion time values.
-
-        The value must be a valid date/time string, and it must be within the
-        server's maximum retention threshold from the dataset's upload time.
+        """Try out some unacceptable values for server.deletion: the value must
+        be a date/time string within the configured maximum lifetime of the
+        dataset upload time.
         """
```

Comment:
Just as an aside, the docstring is supposed to be a brief (e.g., one-line) summary, possibly followed by a blank line and subsequent paragraphs.  While I like the edit you've made here, it doesn't conform to the convention, which the previous text did.  🍤

Reply:
🦐 Most of our unit tests don't have docstrings, and those which do are generally very abbreviated. (Although in a few I went to the length of doing "Args" featuring the fixtures and parametrizations.) Basically, I have trouble caring much about that convention in this context. But if I'm given a strong reason to open this again, I'll think about it.